torch/utils/data/datapipes/iter/loadfilesfromdisk.py

```diff
@@ -1,28 +1,42 @@
+from io import IOBase
+from typing import Iterable, Tuple
+
 from torch.utils.data import IterDataPipe
 from torch.utils.data.datapipes.utils.common import get_file_binaries_from_pathnames
-from typing import Iterable, Iterator, Tuple
-from io import BufferedIOBase
 
-class LoadFilesFromDiskIterDataPipe(IterDataPipe[Tuple[str, BufferedIOBase]]):
+
+class LoadFilesFromDiskIterDataPipe(IterDataPipe[Tuple[str, IOBase]]):
     r""" :class:`LoadFilesFromDiskIterDataPipe`.
 
-    Iterable Datapipe to load file binary streams from given pathnames,
-    yield pathname and binary stream in a tuple.
+    Iterable Datapipe to load file streams from given pathnames,
+    yield pathname and file stream in a tuple.
     args:
         datapipe: Iterable datapipe that provides pathnames
+        mode: An optional string that specifies the mode in which
+            the file is opened. Default 't' refers to text stream
+            and 'b' refers to binary stream.
```

Comment:
Since we internally use [`open`](https://docs.python.org/3/library/functions.html#open) maybe link to it.

Reply:
Good point!